Attribute Analysis.ipynb

```diff
@@ -602,6 +603,340 @@
     "display(ui, out)"
    ]
   },
+  {
+   "source": [
+    "# (M4) Metric: Linearly separable objects by attribute\n",
+    "<a id=\"metric4\"></a>"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## Setup\n",
+    "<a id=\"metric4_setup\"></a>"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hide_toggle(for_next=True, toggle_text='Show/hide M4 code')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
```

Comment:
I added this here to suppress permutation test warnings about the number of items in a class being less than the number of splits.